components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.core/src/main/java/org/wso2/carbon/identity/configuration/mgt/core/constant/SQLConstants.java

```diff
@@ -246,6 +489,58 @@
             "  )\n" +
             "WHERE\n" +
             "  R.ID = ?\n";
+    public static final String GET_RESOURCE_BY_ID_MSSQL_WITHOUT_CREATED_TIME = "SELECT" +
```

Comment:
Other than MySQL, we will not need `without_created_time` queries, as it was introduced to handle backward compatibility when the created time field is not available. Since we support other DBs from this PR onwards, we can decide as the created time field is already available.

Reply:
Please check other places.

Reply:
Removed  `without_created_time` queries written for other database types in  5a0504e41b864eb30d7aa555f60249f9b40e79e9